helm-chart-sources/pulsar/values.yaml

```diff
@@ -1750,6 +1750,21 @@ pulsarHeartbeat:
       topicName: "persistent://public/default/pubsub-latency-test"
   service: {}
 
+# Deploy OpenID (set to false if Keycloak is enabled)
+# For configuring OpenID in Okta for Pulsar, instructions are here: https://www.youtube.com/watch?v=UQBrecHOXxU&ab_channel=DataStaxDevelopers
+# Then, use the values from the Payload generated to populate these fields:
+openid:
+  enabled: false # If true, make sure tokenAuth is disabled.
+  allowedAudience: api://pulsarClient
+  roleClaim: sub
+  attemptAuthenticationProviderToken: true
+  authenticationProvider: com.datastax.oss.pulsar.auth.AuthenticationProviderOpenID
+  # From token generated in Okta UI or other method:
```

Comment:
Maybe we should have a section in the readme describing the relevant okta settings? Also, this is not from the token, this is where users retrieve the token.